app/meetings/index/-components/meetings-list/component.ts

```diff
@@ -10,7 +10,7 @@ export default class MeetingsList extends Component.extend({
 }) {
     // Private properties
     search?: string;
-    sort?: string;
+    sort = '-submissions_count';
```

Comment:
Do we not need to use `defaultTo` here because this value is mutated by an action, instead of by a parameter?

Reply:
Yep!